WebRoot/css/zm.css

```diff
@@ -7597,4 +7590,33 @@ input[type="radio"] + label {
 
 .ZmPrefFilterRow .ZmPrefFilterRowActionsWrapper .ZmFilterRowActions .ZmFilterRowAction {
 	@PrefFilterRowAction@
-}
\ No newline at end of file
+}
+
+/* Appointment tooltip */
+.ZmAppointmentTooltip {
+	@AppointmentTooltip@
+}
+
+.ZmAppointmentTooltip * {
+	@FontSize-slightly-big@
+}
+
+.ZmAppointmentTooltip h3 {
+	@AppointmentTooltipTitle@
+}
+
+.ZmAppointmentTooltip h6 {
```

Comment:
Suggestion, should we be replacing heading elements with div/span and apply sizes using css . Not sure what general practice is ?

Reply:
Before it was using div, span for showing title. I have changed it to heading tag because it represents title and hence more suitable here.

Thanks

Reply:
Thats fine. Just one thing that I wasn't sure was why are we overriding font size for heading tags? They themselves are meant to have different sizes right?

Reply:
Heading tags are not meant to have different size. They meant to have different level of importance semantically. We can style them anyway we want. Let me know if you still have any question.

Reply:
thats fine then. Thanks